pkgpanda/docs/activating.md

```diff
@@ -16,28 +18,34 @@ EnvironmentFile=/opt/mesosphere/config/mesos-slave-config
 ## Activating new packages
 
 1. Validate new list of packages to activate has no conflicts
-   - No conflicting provides
-   - No conflicting systemd names
-   - No conflicting executable names
-   - No conflicting environment variables
-1. Archive old packages
-   - mv active.json active.json.old
-   - rm -rf INSTALL_ROOT/bin INSTALL_ROOT/systemd INSTALL_ROOT/environment INSTALL_ROOT/config
-1. Write new active manifest to active.json.new
-1. Install new package config
-  - Symlink binaries for every pacakge into INSTALL_ROOT/bin
-  - Symlink config for each package into INSTALL_ROOT/config/{provides}/
-  - Aggregate environment variables to INSTALL_ROOT/environment
-1. Enable everthing in systemd
+    - No conflicting provides
+    - No conflicting `systemd` names
+    - No conflicting executable names
+    - No conflicting environment variables
+
+2. Archive old packages
+
+    - `mv active.json active.json.old`
+    - `rm -rf INSTALL_ROOT/bin INSTALL_ROOT/systemd INSTALL_ROOT/environment INSTALL_ROOT/config`
+
+3. Write new active manifest to `active.json.new`
+
+4. Install new package config
+
+    - Symlink binaries for every package into `INSTALL_ROOT/bin`
+    - Symlink config for each package into `INSTALL_ROOT/config/{provides}/`
+    - Aggregate environment variables to `INSTALL_ROOT/environment`
+
+5. Enable everything in `systemd`
```

Comment:
Insert a blank line between 1 and sub-items or remove blank line between 2, 4 and sub-items to be consistent. Otherwise looks good and I find being explicit with the list numbers is generally less error-prone.